src/AzSK.ADO/Framework/Core/STMapping/AzSKADOServiceMapping.ps1

```diff
@@ -11,12 +11,23 @@ class AzSKADOServiceMapping: CommandBase
     [string] $RepositoryMappingsFilePath
     [string] $MappingType
     [string] $OutputFolderPath
+    [string] $Auto = $false
+    [string] $StorageAccount;
+    [string] $StorageRG;
+    [string] $Container;    
+    [object] $StorageAccountCtx;     
+    # Report Storage settings
+    [string] $ReportStorageAccount;
+    [string] $ReportStorageRG;
```

Comment:
Please add comments for your changes
like what is storage RG and what is ReportsStorageRG, right now its not clear why they can be different